testsuite/features/build_validation/init_clients/ubuntu2404_minion.feature

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020-2021 SUSE LLC
```

Comment:
```suggestion
# Copyright (c) 2024 SUSE LLC
```
Since it is a completely new file. Regarding using this specific SUSE copyright, I am unsure. Our PO, @admd will be asking legal about how this should be handled.